packages/textsubsuper/init.lua

```diff
@@ -0,0 +1,366 @@
+      -- figures. We try to ensure we are not using oldstyle numbers...
+      SILE.call("font", {
+        size = ySize,
+        weight = weight == 400 and (weight + SILE.settings:get("textsubsuper.bolder")) or weight,
+        features = "+lnum -onum",
+      }, function ()
+        rescaleContent(content)
+      end)
+    end)
+    SILE.call("kern", { width = -xOffset / 2 })
+  end, "Typeset a fake (raised, scaled) superscript content.")
+
+  SILE.registerCommand("textsubscript:fake", function (_, content)
+    SILE.require("packages/raiselower")
+    local italicAngle = getItalicAngle()
+    local weight = getWeightClass()
+
+    local ratio = SILE.settings:get("textsubsuper.scale")
+    local ySize = ratio * SILE.settings:get("font.size")
+    local yOffset = SILE.settings:get("textsubsuper.offset.subscript")
+    local xOffset = -math.sin(italicAngle * math.pi / 180) * yOffset:absolute()
+    SILE.call("kern", { width = -xOffset })
+    SILE.call("lower", { height = yOffset }, function ()
+      SILE.call("font", {
+        size = ySize,
+        weight = weight == 400 and (weight + SILE.settings:get("textsubsuper.bolder")) or weight,
+        features = "+lnum +onum",
+      }, function ()
+        rescaleContent(content)
+      end)
+    end)
+    SILE.call("kern", { width = xOffset })
+  end, "Typeset a fake (lowered, scaled) subscript content.")
+
+  -- RE-SCALING BY SOME RATIOS
+
+  SILE.registerCommand("textsubsuper:scale", function (options, content)
+    -- Here assume the ouputter to be libtexpdf
+    -- This is supposed to be checked earlier above.
+    local pdf = require("justenoughlibtexpdf")
+
+    local hbox = SILE.call("hbox", {}, content)
+    table.remove(SILE.typesetter.state.nodes) -- Remove the box from queue
+
+    local xRatio, yRatio = options.xRatio, options.yRatio
+    SILE.typesetter:pushHbox({
+      width = hbox.width * xRatio,
+      height = SILE.length(),
+      depth = SILE.length(),
+      outputYourself = function(self, typesetter, line)
+        local X = typesetter.frame.state.cursorX
+        local Y = typesetter.frame.state.cursorY
+        local x0 = X:tonumber()
+        local y0 = -Y:tonumber()
+        typesetter.frame.state.cursorY = Y
+        pdf:gsave()
+        pdf.setmatrix(1, 0, 0, 1, x0, y0)
+        pdf.setmatrix(xRatio, 0, 0, yRatio, 0, 0)
+        pdf.setmatrix(1, 0, 0, 1, -x0, -y0)
+        hbox.outputYourself(hbox, typesetter, line)
+        pdf:grestore()
+        typesetter.frame.state.cursorX = X
+        typesetter.frame.state.cursorY = Y
+        typesetter.frame:advanceWritingDirection(self.width)
+      end
+    })
+  end, "Scale content by some horizontal and vertical ratios")
+end
+
+return {
+  declareSettings = declareSettings,
+  registerCommands = registerCommands,
+  documentation = [[\begin{document}
+\script[src=packages/textsubsuper]% SILE 0.13 needs a big big FIXME, it's 0xDEADBEEF
```

Comment:
I don't understand what you think is the matter here. The manual *partially* loads packages just enough to flesh out a list of valid commands and load documentation. It does *not* initialize packages automatically when outputting their documentation because some of them should not be loaded and would screw up the manual. If you dogfood the package in it's own documentation (as this does by using `\textsubscript` etc. in the documentation) then you have to explicitly use it.

I'll update the API here for 0.14, but if there is something else going on I don't understand let me know.

Reply:
I'll refactor this comment. I can't even remember what it was about (I think the original rebase where I had to add this line occurred in between the time we had `\script` and not yet the new `\use` and things were not that clear about what each would do).

Reply:
Done.